DOCS-1991: Corrections to Lightspeed FAQ texts (#1051)
MultiSafepay/docs: content/integrations/hosted/lightspeed_app/faq/disable-payment-methods.md

meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
---

### Disabling payment methods

> Yes, if you do not want certain payment methods to be displayed at all, you can disable them in the _Payment methods ordering list_. You will see the disabled payment methods in the Payment methods ordering list with a strikethrough.

Reviewer: ```suggestion
If you don't want a payment method to be displayed, you can disable it in the _Payment methods ordering list_. Disabled payment methods are displayed in the ordering list with a strikethrough.
```
